soapui-testserver-api/src/main/java/com/smartbear/ready/recipe/WsdlMockResponseTestStepParser.java

```diff
@@ -0,0 +1,59 @@
+package com.smartbear.ready.recipe;
+
+import com.eviware.soapui.config.MockResponseStepConfig;
+import com.eviware.soapui.config.TestStepConfig;
+import com.eviware.soapui.impl.wsdl.WsdlInterface;
+import com.eviware.soapui.impl.wsdl.WsdlOperation;
+import com.eviware.soapui.impl.wsdl.support.CompressedStringSupport;
+import com.eviware.soapui.impl.wsdl.testcase.WsdlTestCase;
+import com.eviware.soapui.impl.wsdl.teststeps.registry.WsdlMockResponseStepFactory;
+import com.eviware.soapui.support.types.StringToObjectMap;
+import com.smartbear.ready.recipe.teststeps.TestStepStruct;
+import com.smartbear.ready.recipe.teststeps.WsdlMockResponseStepStruct;
+
+import static com.eviware.soapui.support.ModelItemNamer.createName;
+import static com.smartbear.ready.recipe.WsdlUtils.getWsdlInterface;
+
+public class WsdlMockResponseTestStepParser implements TestStepJsonParser {
+    @Override
+    public void createTestStep(WsdlTestCase testCase, TestStepStruct testStepElement, StringToObjectMap context) throws ParseException {
+        WsdlMockResponseStepStruct stepStruct = (WsdlMockResponseStepStruct) testStepElement;
+        WsdlInterface wsdlInterface = getWsdlInterface(testCase, stepStruct.wsdl, stepStruct.binding);
+        if (wsdlInterface == null) {
+            throw new ParseException("Failed to find specified binding [" + stepStruct.binding + "] in WSDL");
```

Comment:
This error message can sometimes be slightly misleading. The problem could be with the WSDL and not with the binding.